src/markdown-pages/docs/writing-own-tests/setting-up-galasa-project.md

```diff
@@ -3,369 +3,191 @@ path: "/docs/writing-own-tests/setting-up-galasa-project"
 
-We are going to build a hierarchy of projects, where the parent project contains:
-
-- A Managers sub-project, allowing you to extend the provided range of Managers. In practice, if you have no intention of writing a Manager, you can omit this - it is included in the sample project for completeness.
+- A Managers sub-project, allowing you to extend the provided range of Managers. In practice, if you have no intention of writing a Manager, you can omit this.
 - An OBR (OSGi Bundle Repository) sub-project, which is mandatory. Galasa uses the OBR to locate your test project(s) and reason about their interdependencies.
-- One or more test sub-projects, that as the name implies, contain the tests themselves. The sample project will contain just one test project.
+- One or more test sub-projects, that as the name implies, contain the tests themselves. 
 
-The parent project establishes all the dependencies for the sub-projects/modules. It builds all the modules in the order of the dependencies - it builds the Manager module before the test projects that use it.
+The parent project establishes all the dependencies for the sub-projects or modules. It builds all the modules in the order of the dependencies - it builds the Manager module before the test projects that use it.
 
 For simplicity, it is assumed that you will only have one version of a test in production at any one time. However, by establishing different versions of your tests, you can have test streams with different versions of the same test project. For the purposes of the forthcoming example, the version of all projects is set to `0.1.0-SNAPSHOT`. The `SNAPSHOT` element forces Galasa to run with the absolute latest copy of your built test project.
 
-## Setting up the filesystem
 
-This setup is designed as an example of how to create a project structure that might ultimately be deployed to a Maven repository so that your Galasa automation system can find everything it needs to run.
+## Building a project hierarchy
+
+In the following example we are going to use the Galasa CLI to build a hierarchy of projects, where the parent project _dev.galasa.example.banking_ contains the following modules: 
+- Two test sub-projects or modules called _dev.galasa.example.banking.payee_ and _dev.galasa.example.banking.account_ 
+- A mandatory OBR sub-project called _dev.galasa.example.banking.obr_.
 
-In a temporary directory - and you can do this with whatever tools you choose, no need to use Eclipse - create the following file structure:
+This setup is designed as an example of how to create a project structure that might ultimately be deployed to a Maven repository, so that your Galasa automation system can find everything it needs to run.
 
+The following example syntax assumes a banking application is being tested. The banking application contains functionality that deals with `payee` and `account` object types, both of which can be manipulated:
+
+```
+galasactl project create \
```

Comment:
> Note: Windows uses '^' instead of `\` so can this be in a separate Windows/Unix/Mac toggle field ? Can we do that ?
Else some call-out for Windows might be good if we can't do that.

Reply:
I have added a new codeblock tag for windows for now